openshift-knative-operator/cmd/operator/kodata/ingress/0.21/3-500-validating-webhook.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: config.webhook.istio.networking.internal.knative.dev
+  labels:
+    serving.knative.dev/release: devel
+    networking.knative.dev/ingress-provider: istio
+webhooks:
+- admissionReviewVersions:
+  - v1beta1
```

Comment:
valid yaml ? (i.e. that's not a key-value pair as part of the arrya's element definition)

Reply:
Not quite parsing the key-value comment. `admissionReviewVersions` is an array of strings.

Reply:
ah, sorry, thought that the `clientConfig:` is on thee same level as `v1beta1`

damned yaml ....